internal/pkg/skuba/kubernetes/node_version.go

```diff
@@ -185,10 +179,10 @@ func nodeVersioningInfoWithClientset(client kubernetes.Interface, nodeName strin
 
 	// find out the container image tags, depending on the role of the node
 	if IsControlPlane(nodeObject) {
```

Comment:
Note that this will probably conflict with #794. We should merge that one first and then rebase this.